src/Components/ReusableComponents/Header/Header.css

```diff
@@ -0,0 +1,39 @@
+.header {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  align-items: center;
+  height: 150px;
```

Comment:
I'd probably give the header a height in relative units as on large or small screens that might cause issues(?).